base/src/main/java/com/smartdevicelink/proxy/rpc/ShowConstantTbt.java

```diff
@@ -49,34 +50,33 @@
  * @see UpdateTurnList
  * @since SmartDeviceLink 2.0
  */
-public class ShowConstantTbt extends RPCRequest {
+public class ShowConstantTBT extends RPCRequest {
 
     public static final String KEY_TEXT1 = "navigationText1";
     public static final String KEY_TEXT2 = "navigationText2";
     public static final String KEY_ETA = "eta";
     public static final String KEY_TOTAL_DISTANCE = "totalDistance";
-    public static final String KEY_MANEUVER_DISTANCE = "distanceToManeuver";
```

Comment:
@kostyaBoss Lines 59-60 and 206-252 are good.

I would revert the other changes, since they are out of scope of the PR.